api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceServiceImpl.java

```diff
@@ -101,10 +102,34 @@ public FireCloudService getFireCloudService() {
 
   @Override
   public Workspace get(String ns, String firecloudName) {
-    return workspaceDao.findByWorkspaceNamespaceAndFirecloudNameAndActiveStatus(
+    Workspace workspace = workspaceDao.findByWorkspaceNamespaceAndFirecloudNameAndActiveStatus(
         ns,
         firecloudName,
         StorageEnums.workspaceActiveStatusToStorage(WorkspaceActiveStatus.ACTIVE));
```

Comment:
Shouldn't this function `workspaceActiveStatusToStorage` need updating? I'm a bit disturbed that it seems our tests didn't fail here - I think active status may not have a test where others do - could you add that as well?

Reply:
Not addressed I think

Reply:
Hmm, my tests didn't catch it again since the value is never actually read by our system. It reads off as a null which, for our purposes, behaves the same way `INACTIVE` should.

I'm not sure if I want to add a null check just to make the test fail but this could be something that gets covered by my hackathon project (adding Kotlin). It enforced non nullability in the type system which can be used in the StorageEnum functions.

Reply:
Not asking for a null check, just want the new value to be included here: https://github.com/all-of-us/workbench/blob/b42c9d552f391a9127a7813fbddbf25e8be026e8/api/src/main/java/org/pmiops/workbench/db/model/StorageEnums.java#L188

(and test case added to ensure that conversion function is exhaustive)

Reply:
I know. I'm adding the new value. I'm just saying its hard to write a test case that actually makes it fail since the null behavior is identical to the INACTIVE behavior.

Reply:
Ah sorry - RE test: to clarify, I was just referring to the missing test case at this level: https://github.com/all-of-us/workbench/blob/b42c9d552f391a9127a7813fbddbf25e8be026e8/api/src/test/java/org/pmiops/workbench/db/model/StorageEnumsTest.java#L24

Though agree would be nice if it were caught at the service level as well.

Reply:
I rewrote the storage enums test to use reflection and auto check all fields. I think it covers everything the old test did.

https://github.com/all-of-us/workbench/pull/2432/commits/1bb93c7125da4be5f6f8b4641b0ca64327c0f30b#diff-dcf82ff88eec81c83dbafd1d56ec62fbR13

Reply:
Example of a test failure

```
name              : org.pmiops.workbench.model.AnnotationType:DATE
expected not to be: null

	at org.pmiops.workbench.db.model.StorageEnumsTest.noMissingMapEntries(StorageEnumsTest.java:37)
```